lib/peerconnection.js

```diff
@@ -18,6 +18,24 @@ function RTCPeerConnection() {
 
   EventTarget.call(this);
 
+  this.getSrcElement = function() {
```

Comment:
Do we really need this? It's not a standard API, so I prefer not to add it.

Reply:
Removed.